SoftLayer/managers/load_balancer.py

```diff
@@ -92,15 +92,30 @@ def update_lb_health_monitors(self, uuid, checks):
     def get_lbaas_uuid_id(self, identifier):
         """Gets a LBaaS uuid, id. Since sometimes you need one or the other.
 
-        :param identifier: either the LB Id, or UUID, this function will return both.
+        :param identifier: either the LB Id, UUID or Name, this function will return UUI and LB Id.
         :return (uuid, id):
         """
-        # int objects don't have a len property.
-        if not isinstance(identifier, int) and len(identifier) == 36:
-            this_lb = self.lbaas.getLoadBalancer(identifier, mask="mask[id,uuid]")
+        mask = "mask[id,uuid]"
+        if isinstance(identifier, int):
```

Comment:
Need to also check if the string we got is really just a number. [isdigit](https://docs.python.org/3/library/stdtypes.html#str.isdigit) should be exactly what we need.

`if isinstance(identifier, int) or indentifier.isdigit(): `


```
$ ./slcli -v  lb detail 874328
Calling: SoftLayer_Network_LBaaS_LoadBalancer::getAllObjects(id=None, mask='mask[id,uuid]', filter='{'name': {'operation': '874328'}}', args=(), limit=None, offset=None))
Calling: SoftLayer_Network_LBaaS_LoadBalancer::getObject(id=None, mask='mask[healthMonitors, l7Pools, members, sslCiphers, listeners[defaultPool[healthMonitor,members, sessionAffinity],l7Policies]]', filter='None', args=(), limit=None, offset=None))
SoftLayerAPIError(500): Object does not exist to execute method on. (SoftLayer_Network_LBaaS_LoadBalancer::getObject)

```